lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -32,20 +41,19 @@ def test_invisible_axes(fig_test, fig_ref):
 
 @mpl3d_image_comparison(['aspects.png'], remove_text=False)
 def test_aspects():
-    aspects = ('auto', 'equal', 'equalxy', 'equalyz', 'equalxz')
-    fig, axs = plt.subplots(1, len(aspects), subplot_kw={'projection': '3d'})
+    aspects = ('auto', 'equal', 'equalxy', 'equalyz', 'equalxz', 'equal')
+    _, axs = plt.subplots(2, 3, subplot_kw={'projection': '3d'})
 
-    # Draw rectangular cuboid with side lengths [1, 1, 5]
-    r = [0, 1]
-    scale = np.array([1, 1, 5])
-    pts = itertools.combinations(np.array(list(itertools.product(r, r, r))), 2)
-    for start, end in pts:
-        if np.sum(np.abs(start - end)) == r[1] - r[0]:
-            for ax in axs:
-                ax.plot3D(*zip(start*scale, end*scale))
-    for i, ax in enumerate(axs):
+    for ax in axs.flatten()[0:-1]:
+        plot_cuboid(ax, scale=[5, 1, 1])
+    # plot a cube as well to cover github #25443
+    plot_cuboid(axs[1][2], scale=[1, 1, 1])
```

Comment:
Over here. The first five subplots are done with 5x1x1 cuboid, but then the last one is done as a 1x1x1 regular cube. What I'm saying is that the first row can be done as 5x1x1, and the second row as 1x1x1, with each column being a different aspect mode.

We also still have to explain why some of the original five plots now look different. Was that supposed to be expected? I want to make sure the results make sense.

Reply:
Ah originally it was a 1x1x5 and got switched as I was iterating, I switched it back to that just now so it should match what was there before.

I don't see much value in the second row... the cube was added because that was a failure case in the linked issue, but I don't think it help too much to demonstrate the behavior to whoever is looking at tests rather than docs.
![aspects](https://user-images.githubusercontent.com/14363975/227097092-1641c733-5ba3-498b-94c3-d7838635458b.png)

Reply:
Yeah, I think you are right, it isn't really all that illuminating. What you have now is good.